app/actions/TrezorActions.js

```diff
@@ -111,20 +114,26 @@ export const TRZ_SELECTEDDEVICE_CHANGED = "TRZ_SELECTEDDEVICE_CHANGED";
 export const TRZ_NOCONNECTEDDEVICE = "TRZ_NOCONNECTEDDEVICE";
 
 function onChange(dispatch, getState, features) {
-  if (features == null) throw "no features on change";
+  if (features == null) return;
```

Comment:
Do the new versions not return a `features` object in boot loader mode? Why are you changing these from throws (which should ultimately at least log the error) to simple returns?

Reply:
This probably didn't need to be changed. Reverting.